### ISSUE TITLE
Rotate the image and keep metadata but update/remove EXIF orientation data

### ISSUE THREAD

**Issue by andriystabryn** [blueimp/JavaScript-Load-Image#96]
Hi there! 
I'm trying to rotate the picture based on Orientation but save all meta attributes
```javascript
loadImage.parseMetaData(
        file,
        function(data) {
            loadImage(file, img => {
              img.toBlob(blob => {
                var blobWithMeta = new Blob(
                    [
                        data.imageHead,
                        blob
                    ],
                    { type: blob.type }
                  );
                resolve(blobWithMeta);
              }, 'image/jpeg');
            }, {
              orientation: true
            });
        },
    );
```

But the final image still has Orientation meta attribute. How can I remove just one meta from the `data.imageHead`?

**Comment by blueimp** [comment]
As of now this library does not support writing individual EXIF attributes.
I've marked this as a feature request.

**Comment by Nikhil144** [comment]
with this code  image size getting increase by 4-5 times than original image.
How should i keep it to original size or near to it?

**Comment by andriystabryn** [comment]
@Nikhil144 the functionality to keep metadata and rotate the picture is unavailable now. It's only feature request

**Comment by blueimp** [comment]
I've updated the title to make the intention of this feature clearer.

**Comment by blueimp** [comment]
@Nikhil144 If you have an issue that you can reproduce, please create a separate issue ticket.
This ticket here is about adding support to write updated orientation data into the image meta data header.

Regarding iPhone device support, this should not be dependent on the device form factor, but on the supported iOS version and therefore the supported Safari version. iPhone 6s supports the latest (v13) iOS and therefore has support for the orientation feature in this library.

**Comment by blueimp** [comment]
I'e released `blueimp-load-image@2.31.0` which introduces minimal Exif writer support:
https://github.com/blueimp/JavaScript-Load-Image#exif-writer

**Comment by Nikhil144** [comment]
Thank you very much for that . Thats very helpful .

Thanks  its very cool  !!
